pep-0394.txt

```diff
@@ -187,6 +212,16 @@ attempting to execute a script containing Python 2 specific syntax with a
 Python 3 interpreter.
 
 
+Forward Compatibility
+=====================
+
+A problem can arise if a script compatible with both Python 2 and 3, which
+has ``python`` in its shebang line, is invoked on a system that does not have
+Python 2 (and thus, the ``python`` command) installed.
+This is mostly a non-issue -- as with any other case of a required interpreter
+not being installed, the sysadmin can install the ``python`` command.
```

Comment:
Again, I don't want sysadmins to feel empowered by this. Assuming the shebang line uses `#!/bin/env python` this can be remedied by activating a venv.